### ISSUE TITLE
matching in verbpdgm27 Present System Middle-Passive of Contract Verbs in -όω table

### ISSUE THREAD

**Issue by monzug** [alpheios-project/inflection-tables#334]
we do not have a match on the table for   δηλοῖ    2nd person sing subjective  in Present System Middle-Passive of Contract Verbs in -όω table. it's indicative mp but not subjective. I did try combinations of different accents/endings with not much luck.
also, the duals  optative do not have a match:
δηλοίσθην goes to imperative w_stem stemtype   and not to  ww_pr stemtype
δηλοῖσθον  lexical query produced no results
Vanessa said that duals optative are very rare.

**Comment by vgorman1** [comment]
Diogenes lists this:
δηλοί: (pres subj mp 2nd sg) (pres ind mp 2nd sg) (pres subj act 3rd sg)
δηλοῖ: (pres ind mp 2nd sg) (pres opt act 3rd sg) (pres ind act 3rd sg)

Smyth uses δηλόω as a paradigm verb, so all the normal forms are there. The one with the circumflex is listed in Smyth for both these forms, so that would be the more usual form.
δηλοίσθην is, according to Smyth the 3rd person dual and δηλοῖσθον is the 2nd person dual but, according to Diogenes, neither ever occurs.

Vanessa B. Gorman
Professor, Ancient History
Undergraduate Chair
Dept. of History
University of Nebraska-Lincoln
619 Oldfather Hall
Lincoln, NE 68588-0327

**Comment by monzug** [comment]
Looking at the tables from 166 to 171 on Smyth, I confirm that δηλοῖ  should be 2nd person singular indicative mp and subjective mp. Morpheus is missing this.
also, the duals optative should be δηλοῖσθον and δηλοίσθην. Morpheus is missing one and the other has wrong stemtype

**Comment by vgorman1** [comment]
δηλοῖ can be:
δηλόω (make visible): pres ind mp 2nd sg
δηλόω (make visible): pres opt act 3rd sg
δηλόω (make visible): pres ind act 3rd sg


δηλοί: (pres subj mp 2nd sg)

The accent is different.

The dual optatives never occur.

Vanessa B. Gorman
Professor of Ancient History
Department of History
619 Oldfather Hall
University of Nebraska-Lincoln
Lincoln, NE 68588-0327
https://vgorman1.github.io/

**Comment by monzug** [comment]
Vanessa, I see what you mean. However in Smith (a grammar of the greek language) and Smyth and in Mastronarde tables, the indicative and subjective is δηλοῖ. in Morpheus, we have δηλοῖ and δηλοί as you suggested.
Should we change as per Morpheus and Diogenes or still follow Smyth (and in this case also Smith and Mastronarde)?
as per the duals, again they are in Smith, Smyth and Mastronarde but not in Morpheus. Should we remove the entries from the table? I would keep them.

**Comment by vgorman1** [comment]
You are correct.  Always follow Smyth first.  He has been around so very long that I would venture to say that typos have long since been eliminated.   I think the 3rd person label must have been a mistake in Morpheus.

Vanessa B. Gorman
Professor of Ancient History
Department of History
619 Oldfather Hall
University of Nebraska-Lincoln
Lincoln, NE 68588-0327
https://vgorman1.github.io/

**Comment by vgorman1** [comment]
If forms are in Smyth, that means he created them from the rules.  He didn’t have access to computers to know if they ever existed or not.  Since he was a very learned man, if they occur in Smyth, go with them!  He is undoubtedly correct.

Vanessa B. Gorman
Professor of Ancient History
Department of History
619 Oldfather Hall
University of Nebraska-Lincoln
Lincoln, NE 68588-0327
https://vgorman1.github.io/

**Comment by monzug** [comment]
Vanessa, merci'!!!
